src/page-building/career/culture.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import styled from 'styled-components';
+import { up } from '../../components/breakpoint/breakpoint';
+import { Teaser } from '../../components/teasers/teaser';
+import { Illustration } from '../../components/illustration/illustration';
+import { IllustrationType } from '../../components/illustration/illustration-set';
+import { SectionHeader } from '../../new-components/section-header/section-header';
+
+interface CultureAspect {
+  illustration: IllustrationType;
+  title: string;
+  description: string;
+}
+
+const TeaserGrid = styled.div`
```

Comment:
I copied this from index.tsx and I think we should provide such a grid from within the teaser component package to share them properly 👍